### PR TITLE
Documents new merge success metrics 

### PR DESCRIPTION
**Pull request by iAmWillShepherd**
🌵 Depends on #5810 🌵 

This PR documents the new metrics requested in #5394: `mergeSuccessAfterConflictsCount` and `mergeAbortedAfterConflictsCount`.

### CODE REVIEW

File: docs/process/metrics.md [hunk 1]
```diff
@@ -53,3 +53,5 @@ These are general metrics about feature usage and specific feature behaviors. Th
 | `mergedWithLoadingHintCount` | The number of times the user merged before seeing the result of the merge hint. | To understand how many people are merging before learning whether there will be conflicts or not |
 | `mergedWithCleanMergeHintCount` | The number of times the user has merged after seeing the 'no conflicts' merge hint. | To understand how many "clean" merges there are |
 | `mergedWithConflictWarningHintCount` | The number of times the user has merged after seeing the 'you have XX conflicted files' warning. | To understand how frequently people are merging even though they know there will be conflicts |
+| `mergedSuccesfulAfterConflictsCount` | The number of times the user successfully completes a merge after a merge conflict. | To understand how effectively users are able to resolve conflicts and complete their merge successfully |
+| `mergesAbortedAfterConflictsCount` | The number of times the user aborts a merge after a merge conflict. | To understand the frequency of merges that are never completed after attempting to merge and hitting a merge conflict |
```
Comment (shiftkey) [inline docs/process/metrics.md]: Is there a reason for the `s` in `merges` as well as `Count`? Could we just use `mergeAbortedAfterConflictsCount` for the key?

File: docs/process/metrics.md [hunk 2]
```diff
@@ -53,3 +53,5 @@ These are general metrics about feature usage and specific feature behaviors. Th
 | `mergedWithLoadingHintCount` | The number of times the user merged before seeing the result of the merge hint. | To understand how many people are merging before learning whether there will be conflicts or not |
 | `mergedWithCleanMergeHintCount` | The number of times the user has merged after seeing the 'no conflicts' merge hint. | To understand how many "clean" merges there are |
 | `mergedWithConflictWarningHintCount` | The number of times the user has merged after seeing the 'you have XX conflicted files' warning. | To understand how frequently people are merging even though they know there will be conflicts |
+| `mergedSuccesfulAfterConflictsCount` | The number of times the user successfully completes a merge after a merge conflict. | To understand how effectively users are able to resolve conflicts and complete their merge successfully |
```
Comment (shiftkey) [inline docs/process/metrics.md]: I know we haven't specified these to be gramatically correct but `mergedSuccessful` is messing with me. Could this be `mergedSuccessfullyAfterConflictsCount`? Or maybe just `mergeSuccessAfterConflictsCount`?

Comment (j-f1) [reply docs/process/metrics.md]: `successfulConflictResolutionCount`?

**Comment by iAmWillShepherd** [comment]
🍏

**Review by shiftkey — APPROVED** [review]

**Comment by shiftkey** [comment]
As this isn't targeting `master` I plan to merge this in the morning (my time) to give everyone else a chance to look over the new names.

**Comment by budkamLukasz** [comment]
> 🌵 Depends on #5810 🌵
> 
> This PR documents the new metrics requested in #5394: `mergeSuccessAfterConflictsCount` and `mergeAbortedAfterConflictsCount`.

**Pull request merged**